Build task syncNoticeAndLicense fails on Windows 11

### Describe the bug

While building the polaris from the documentation 
https://polaris.apache.org/in-dev/unreleased/getting-started/quickstart/

We are on windows 11 
Java version 
java -version
java version "21.0.8" 2025-07-15 LTS
Java(TM) SE Runtime Environment (build 21.0.8+12-LTS-250)
Java HotSpot(TM) 64-Bit Server VM (build 21.0.8+12-LTS-250, mixed mode, sharing)

And the cmd we are running and the error we are getting

 ./gradlew   :polaris-server:assemble   :polaris-server:quarkusAppPartsBuild --rerun   :polaris-admin:assemble   :polaris-admin:quarkusAppPartsBuild --rerun   -Dquarkus.container-image.build=true

Starting a Gradle Daemon, 1 busy and 2 incompatible and 8 stopped Daemons could not be reused, use --status for details
Configuration on demand is an incubating feature.

  

> > Task :polaris-api-management-model:openApiGenerate
>   user-defined server variable support is experimental.
>   Unexpected allVars for AwsIamServiceIdentityInfo expecting:2 vars. actual:1 vars
>   Unexpected allVars for BearerAuthenticationParameters expecting:2 vars. actual:1 vars
>   Unexpected allVars for HadoopConnectionConfigInfo expecting:5 vars. actual:4 vars
>   Unexpected allVars for HiveConnectionConfigInfo expecting:5 vars. actual:4 vars
>   Unexpected allVars for IcebergRestConnectionConfigInfo expecting:5 vars. actual:4 vars
>   Unexpected allVars for OAuthClientCredentialsParameters expecting:5 vars. actual:1 vars                                                     
>   Unexpected allVars for SigV4AuthenticationParameters expecting:6 vars. actual:1 vars                                                        
>   
>   > Task :polaris-api-management-service:openApiGenerate                                                                                      
>   user-defined server variable support is experimental.
>   ################################################################################                                                            
>    Thanks for using OpenAPI Generator.                                          #
>    Please consider donation to help us maintain this project ≡ƒÖÅ                 #
>    https://opencollective.com/openapi_generator/donate                          #
>   ################################################################################
>   Successfully generated code to E:\Work-YDC\Tasks\polaris\api\management-service\build\generated-openapi
>   
>   > Task :polaris-api-management-model:openApiGenerate
>   ################################################################################
>    Thanks for using OpenAPI Generator.                                          #
>    Please consider donation to help us maintain this project ≡ƒÖÅ                 #
>    https://opencollective.com/openapi_generator/donate                          #
>   ################################################################################
>   Successfully generated code to E:\Work-YDC\Tasks\polaris\api\management-model\build\generated-openapi
>   
>   > Task :polaris-version:syncNoticeAndLicense FAILED
>   
>   > Task :polaris-api-iceberg-service:openApiGenerate
>   user-defined server variable support is experimental.
>   ################################################################################
>    Thanks for using OpenAPI Generator.                                          #                                                            
>    Please consider donation to help us maintain this project ≡ƒÖÅ                 #
>   https://opencollective.com/openapi_generator/donate                          #
>   ################################################################################
>   Successfully generated code to E:\Work-YDC\Tasks\polaris\api\iceberg-service\build\generated-openapi                                        
>   
>   > Task :polaris-api-catalog-service:openApiGenerate
>   
> 
> > user-defined server variable support is experimental.
> >   Failed to get the schema name: null
> >   ################################################################################
> >    Thanks for using OpenAPI Generator.                                          #
> >    Please consider donation to help us maintain this project ≡ƒÖÅ                 #
> >    https://opencollective.com/openapi_generator/donate                          #
> >   #####
> 
>    Successfully generated code to E:\Work-YDC\Tasks\polaris\api\polaris-catalog-service\build\generated-openapi
>   
>  >FAILURE: Build failed with an exception.
>   
>   * What went wrong:
>   Execution failed for task ':polaris-version:syncNoticeAndLicense'.
>   > Cannot access input property '$1' of task ':polaris-version:syncNoticeAndLicense'. Accessing unreadable inputs or outputs is not supported. Declare the task as untracked by using Task.doNotTrackState(). For more information, please refer to https://docs.gradle.org/8.14.3/userguide/incremental_build.html#sec:disable-state-tracking in the Gradle documentation.
>      > Could not stat file E:\Work-YDC\Tasks\polaris\NOTICE*
>   
>   * Try:
>   > Run with --stacktrace option to get the stack trace.
>   > Run with --info or --debug option to get more log output.
>   > Run with --scan to get full insights.
>   > Get more help at https://help.gradle.org.
>   
>   BUILD FAILED in 1m 46s
>   25 actionable tasks: 10 executed, 2 from cache, 13 up-to-date

  
### To Reproduce

1.  Clone the repo
2. run the given cmd in git bash

### Actual Behavior

_No response_

### Expected Behavior

_No response_

### Additional context

_No response_

### System information

_No response_

--- a/tools/version/build.gradle.kts
+++ b/tools/version/build.gradle.kts
@@ -32,9 +32,9 @@ description =
 val syncNoticeAndLicense by
   tasks.registering(Sync::class) {
     // Have to manually declare the inputs of this task here on top of the from/include below
-    inputs.files(rootProject.fileTree(rootProject.rootDir) {
-      include("NOTICE*", "LICENSE*", "version.txt")
-    })
+    inputs.files(
+      rootProject.fileTree(rootProject.rootDir) { include("NOTICE*", "LICENSE*", "version.txt") }
+    )
     inputs.property("version", project.version)
     destinationDir = project.layout.buildDirectory.dir("notice-licenses").get().asFile
     from(rootProject.rootDir) {

--- a/tools/version/build.gradle.kts
+++ b/tools/version/build.gradle.kts
@@ -32,7 +32,9 @@ description =
 val syncNoticeAndLicense by
   tasks.registering(Sync::class) {
     // Have to manually declare the inputs of this task here on top of the from/include below
-    inputs.files(rootProject.layout.files("NOTICE*", "LICENSE*", "version.txt"))
+    inputs.files(rootProject.fileTree(rootProject.rootDir) {
+      include("NOTICE*", "LICENSE*", "version.txt")
+    })
     inputs.property("version", project.version)
     destinationDir = project.layout.buildDirectory.dir("notice-licenses").get().asFile
     from(rootProject.rootDir) {

